### ISSUE TITLE
GFLOPs per forward pass

### ISSUE THREAD

**Issue by Ankowa** [FoundationVision/VAR#129]
Hi,
is there an estimate of GFLOPs per forward pass of the model? I'm looking for an information similar to the one in the DiT paper (https://arxiv.org/pdf/2212.09748).
Best,
Antoni Kowalczuk

**Comment by Ankowa** [comment]
Ok, nevermind, here's how to do it
import torch
from torchprofile import profile_macs

images = torch.randn(1, 3, 256, 256).to(device)
labels = torch.randint(0, 1000, (1,)).to(device)

prof = profile_macs(model, (images, labels))

print(prof)
